<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoRunViewerClass</class>
 <widget class="QMainWindow" name="AutoRunViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoRunViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="logonTab">
       <attribute name="title">
        <string>Logon</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
      <widget class="QWidget" name="serviceTab">
       <attribute name="title">
        <string>Service</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="QWidget" name="driverTab">
       <attribute name="title">
        <string>Driver</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QWidget" name="taskTab">
       <attribute name="title">
        <string>Tasks</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QWidget" name="dllTab">
       <attribute name="title">
        <string>KnownDLLs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutoRunViewer.qrc"/>
 </resources>
 <connections/>
</ui>
